R/quosure-constructor.R

```diff
@@ -9,12 +9,11 @@
 #' @param env (`environment`) Environment being a result of the `code` evaluation.
 #'
 #' @examples
-#' new_quosure(env = list2env(list(a = 1)), code = "a <- 1")
```

Comment:
this should stay as we are still supporting chars but change the implementation only
you can actually reorder to showoff that expression is preferred compared to char